frameworks/g-cloud-9/questions/services/dataSanitisationTypeSoftware.yml

```diff
@@ -0,0 +1,18 @@
+id: dataSanitisationType
+name: Data sanitisation type
+question: Do you have a data sanitisation process?
+
+depends:
+  - "on": lot
+    being:
+      - cloud-software
+
+type: radios
```

Comment:
Yeah, have wrapped it in 'Do you have a data sanitisation process?', made it checkbox and removed the 'No data sanitisation process' option. We could reword it to be something about primary form of data sanitisation but would be a bit open to interpretation.